### PR TITLE
Mailing dependency injection

### PR DESCRIPTION
**Pull request by ruxmur**

### CODE REVIEW

**Review by egorbwork — CHANGES_REQUESTED** [review]
Please finish till the end required changes.

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,5 @@ npm-debug.log
 yarn-error.log
 /.idea
 /.vscode
+
+_ide_hepler.php
```
[the remark this answers is not in the document]
Comment (ruxmur) [reply .gitignore:17]: I watched the video again, and did all the steps like you did. and the file was removed

File: resources/views/emails/contactUs.blade.php [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
   <p>User requested attention:</p>
   <p>User firstName: {{ $firstName }}</p>
   <p>User lastName: {{ $lastName }}</p>
   <p>User email: {{ $email }}</p>
   <p>User tel: {{ $tel }}</p>
-  <p>User sex: {{ $sex }}</p>
-  <p>User membership: {{ $membership }}</p>
-  <p>User sport: {{ $sport }}</p>
-  <p>User coach: {{ $coach }}</p>
+  <p>Sex interested in: {{ implode(',', $sex ?? []) }}</p>
```
Comment (egorbwork) [inline resources/views/emails/contactUs.blade.php:16]: It looks pretty strange. Please explain your intention.

**Review by egorbwork — APPROVED** [review]
The required logic looks workable, so I approved. But I really do not understand idea in the new email texts.

File: resources/views/emails/contactUs.blade.php (hunk 2 above)
Comment (ruxmur) [reply resources/views/emails/contactUs.blade.php]: Changed.

**Pull request merged**